<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Augmentation</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>710</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>661</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox_normalize_enabled">
      <property name="text">
       <string>Normalize</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_mean">
      <property name="text">
       <string>Mean</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_mean1"/>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_mean2"/>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_mean3"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_std">
      <property name="text">
       <string>Std</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_std1"/>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_std2"/>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_std3"/>
      </item>
     </layout>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="spinBox_center_crop_image_size_single"/>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="checkBox_center_crop_enabled">
      <property name="text">
       <string>Center Crop</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QCheckBox" name="checkBox_resize_enabled">
      <property name="text">
       <string>Resize</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QCheckBox" name="checkBox_resize_image_size_single_enabled">
      <property name="text">
       <string>Single Size</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QCheckBox" name="checkBox_resize_image_size_wh_enabled">
      <property name="text">
       <string>Width / Height</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QSpinBox" name="spinBox_resize_image_size_single"/>
    </item>
    <item row="10" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSpinBox" name="spinBox_resize_image_width"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_resize_image_height"/>
      </item>
     </layout>
    </item>
    <item row="12" column="0">
     <widget class="QCheckBox" name="checkBox_random_horizontal_flip_enabled">
      <property name="text">
       <string>Random Horizontal Flip</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QCheckBox" name="checkBox_random_vertical_flip_enabled">
      <property name="text">
       <string>Random Vertical Flip</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QCheckBox" name="checkBox_color_jitter_enabled">
      <property name="text">
       <string>Color Jitter</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QCheckBox" name="checkBox_random_rotation_enabled">
      <property name="text">
       <string>Random Rotation</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="11" column="0">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="17" column="0">
     <widget class="QLabel" name="label_random_rotation_angle">
      <property name="text">
       <string>Angle</string>
      </property>
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QSpinBox" name="spinBox_random_rotation_angle"/>
    </item>
    <item row="15" column="0">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="25" column="0">
     <widget class="QCheckBox" name="checkBox_grayscale_enabled">
      <property name="text">
       <string>Grayscale</string>
      </property>
     </widget>
    </item>
    <item row="24" column="0">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="19" column="0">
     <widget class="QCheckBox" name="checkBox_random_resized_crop_enabled">
      <property name="text">
       <string>Random Resized Crop</string>
      </property>
     </widget>
    </item>
    <item row="20" column="0">
     <widget class="QCheckBox" name="checkBox_random_resized_crop_image_size_single_enabled">
      <property name="text">
       <string>Single Size</string>
      </property>
     </widget>
    </item>
    <item row="20" column="1">
     <widget class="QSpinBox" name="spinBox_random_resized_crop_image_size_single"/>
    </item>
    <item row="21" column="0">
     <widget class="QCheckBox" name="checkBox_random_resized_crop_image_size_wh_enabled">
      <property name="text">
       <string>Width / Height</string>
      </property>
     </widget>
    </item>
    <item row="21" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QSpinBox" name="spinBox_random_resized_crop_image_width">
        <property name="maximum">
         <number>99</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_random_resized_crop_image_height"/>
      </item>
     </layout>
    </item>
    <item row="22" column="0">
     <widget class="QLabel" name="label_random_resized_crop_scale">
      <property name="text">
       <string>Scale</string>
      </property>
     </widget>
    </item>
    <item row="23" column="0">
     <widget class="QLabel" name="label_random_resized_crop_ratio">
      <property name="text">
       <string>Ratio</string>
      </property>
     </widget>
    </item>
    <item row="22" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_random_resized_crop_scale1"/>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_random_resized_crop_scale2"/>
      </item>
     </layout>
    </item>
    <item row="23" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_random_resized_crop_ratio1"/>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_random_resized_crop_ratio2"/>
      </item>
     </layout>
    </item>
    <item row="18" column="0">
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="checkBox_center_crop_image_size_single_enabled">
      <property name="text">
       <string>Single Size</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="checkBox_center_crop_image_size_wh_enabled">
      <property name="text">
       <string>Width / Height</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QSpinBox" name="spinBox_center_crop_image_width"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_center_crop_image_height"/>
      </item>
     </layout>
    </item>
    <item row="12" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_random_horizontal_flip_probability"/>
    </item>
    <item row="13" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_random_vertical_flip_probability"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
